ksql-common/src/main/java/io/confluent/ksql/schema/ksql/FormatOptions.java

```diff
@@ -28,6 +28,10 @@ public static FormatOptions none() {
     return new FormatOptions(word -> true);
   }
 
+  public static FormatOptions noEscape() {
```

Comment:
Probably worth a java doc that this is dangerous to use in anything other than in feedback to the user, e.g. error messages

Reply:
will do this!